app/src/main/cpp/src/components/transport_manager/src/bluetooth_le/bluetooth_le_transport_adapter.cc

```diff
@@ -63,12 +60,21 @@ BluetoothLeTransportAdapter::BluetoothLeTransportAdapter(
           new BluetoothLeConnectionFactory(this),
           NULL,
           last_state_wrapper,
-          settings) 
+          settings)
+          , ble_server_(new BleServer("./localBleWriter"))
+          , ble_control_server_(new BleControlServer("./localBleControl"))
```

Comment:
Suppose to create separate constants for these addresses

Reply:
done in 66d56ca414ac2d17dea823a2186a8d3f2569ec83